### ISSUE TITLE
Change Toast Properties In-Flight

### ISSUE THREAD

**Issue by jadar** [cruffenach/CRToast#161]
Is there a way to change a property, while a notification is in fligh? For example, changing the outgoing animation based on user interaction.

**Comment by dmiedema** [comment]
In theory I think you could if you got the `CRToast.options` dictionary since thats where all the information is kept. It doesn't support that kind of thing right now but I could definitely see that being nice especially for something like a swipe to the side dismissal type of thing

**Comment by jadar** [comment]
Yes. So I could have like user-dismissed progress notification will animate to the left to signify going into a hidden progress indicator and animating up meand done.

That might work. I'll think about it.

**Comment by jadar** [comment]
Yes, that solution work out alright.

**Comment by jadar** [comment]
Wrong issue.

**Comment by chenzhijie** [comment]
in CRToastManager.h, CRToastManager does not have a property or selector like "manager". How can I perform [CRToastManager manager]

**Comment by dmiedema** [comment]
You could add a that to the category on `CRToastManager` also

``` objc
@interface CRToastManager (Private)
+ (instancetype)manager;
@end
```
Forgot that's not a public property

**Comment by jadar** [comment]
This isn't a very good solution, as it will break as soon as CRToast changes the method name.. A bit like calling private methods using selectors..

I'd almost rather see "interaction" blocks that pass the toast (which exposes the options dictionary) they're from as a parameter when they're called.
